api/v1/models/backend_address.go

```diff
@@ -39,6 +39,9 @@ type BackendAddress struct {
 	// State of the backend for load-balancing service traffic
 	// Enum: [active terminating quarantined maintenance]
 	State string `json:"state,omitempty"`
+
+	// Backend weight
+	Weight uint16 `json:"weight,omitempty"`
```

Comment:
I just realized: doesn't this need to be a pointer, since 0 is a valid value?

Reply:
you mean to distinguish between unset and 0 value?

Reply:
exactly :-)